plugins/inputs/vsphere/sample.conf

```
@@ -136,6 +136,12 @@
   datacenter_metric_exclude = [ "*" ] ## Datacenters are not collected by default.
   # datacenter_instances = false ## false by default
 
+  ## VSAN
+  vsan_metric_include = [] ## if omitted or empty, all metrics are collected
+  vsan_metric_exclude = [ "*" ] ## vSAN are not collected by default.
+  ## Whether to skip verifying vSAN metrics against the ones from GetSupportedEntityTypes API.
+  vsan_metric_skip_verify = false ## false by default.
```

Default values should be commented out..
```suggestion
  # vsan_metric_skip_verify = false ## false by default.
```